ang/test/mocks/services/formatCase.mock.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env jasmine */
+
+(function () {
+  var module = angular.module('civicase');
+
+  module.factory('formatCase', function () {
+    return jasmine.createSpy('formatCase')
+      .and.callFake(function (caseObj) {
```

Comment:
Could you remind me why cant we use the original `formatCase`?

Reply:
see [this comment](https://github.com/compucorp/uk.co.compucorp.civicase/pull/113#discussion_r232708012)